tvx/examine.go

```diff
@@ -14,7 +14,8 @@ import (
 	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
```

Comment:
should we leave it as a 'todo' to migrate these specs-actors' references to the stable versions?

Reply:
Oh good point, this needs to be migrated!